### PR TITLE
Don't set `x-single-active-consumer` for streams

### PR DESCRIPTION
**Pull request by rjgildea**
The queue declaration will fail if we do.

Check response status code to catch failed api calls

### CODE REVIEW

File: src/zocalo/cli/configure_rabbitmq.py [hunk 1]
```diff
@@ -359,31 +364,36 @@ def run():
...
+        for b in api.bindings():
+            q = [
+                qu for qu in queues if qu.vhost == b.vhost and qu.name == b.destination
+            ][0]
+            if q.auto_delete or q.exclusive:
+                continue
+            permanent_bindings.append(b)
+        update_config(api, binding_specs, permanent_bindings)
+    except requests.exceptions.HTTPError as e:
+        logger.error(e)
```
Comment (d-j-hatton) [inline src/zocalo/cli/configure_rabbitmq.py:396]: Do we want `exc_info=True`?

Comment (rjgildea) [reply src/zocalo/cli/configure_rabbitmq.py:396]: I thought that in the context of the configuration script that the traceback didn't really add any additional information - the important details come in the error message returned by the API. E.g. attempting to create a binding between a queue `bar` and non-existent exchange `foo` gives:
```
$ zocalo.configure_rabbitmq /path/to/rabbitmq-configuration.yaml 
Creating queue: name='bar' durable=True auto_delete=False arguments={'x-queue-type': 'quorum', 'x-single-active-consumer': False} vhost='zocalo'
creating BindingSpec source='foo' destination='bar' destination_type=<DestinationType.QUEUE: 'q'> vhost='zocalo' routing_key='bar' arguments={}
404 Client Error: Not Found for url: http://example.com:15672/api/bindings/zocalo/e/foo/q/bar
```
I'm not convinced that adding the relevant traceback adds any useful information here...

Comment (d-j-hatton) [reply src/zocalo/cli/configure_rabbitmq.py:396]: Yeah, ok. If you think it won't add anything that's fine

**Comment by d-j-hatton** [comment]
Out of interest what are you thinking of using streams for?

**Pull request merged**